rasgotransforms/rasgotransforms/testing_utils.py

```diff
@@ -0,0 +1,131 @@
+from pathlib import Path
+import os
+import inspect
+import json
+from typing import List
+
+import pandas as pd
+from snowflake import connector as sf_connector
+from dotenv import load_dotenv
+from jinja2.nodes import Macro
+
+from rasgotransforms.main import _load_all_yaml_files, _parse_transform_args_from_yaml, TransformTemplate
+from rasgotransforms.render import RasgoEnvironment
+
+DOTENV_PATH = Path(os.path.dirname(__file__)).parent.parent / ".env"
+load_dotenv(DOTENV_PATH)
+
+TRANSFORMS_DIR = Path(os.path.dirname(__file__)) / "transforms"
+MACROS_DIR = Path(os.path.dirname(__file__)) / "macros"
+SNOWFLAKE_ACCOUNT = os.environ.get('SNOWFLAKE_ACCOUNT')
+SNOWFLAKE_WAREHOUSE = os.environ.get('SNOWFLAKE_WAREHOUSE')
+SNOWFLAKE_USER = os.environ.get('SNOWFLAKE_USER')
+SNOWFLAKE_PASSWORD = os.environ.get('SNOWFLAKE_PASSWORD')
+SNOWFLAKE_ROLE = os.environ.get('SNOWFLAKE_ROLE')
+ARTIFACTS_DIR = os.environ.get('ARTIFACTS_DIR')
+
+
+CREDS = {
+    "account": SNOWFLAKE_ACCOUNT,
+    "warehouse": SNOWFLAKE_WAREHOUSE,
+    "user": SNOWFLAKE_USER,
+    "password": SNOWFLAKE_PASSWORD,
+    "role": SNOWFLAKE_ROLE,
+}
+
+
+def query_into_dataframe(query) -> pd.DataFrame:
```

Comment:
I don't see these called anywhere in the test scripts (which is good), assuming they're called by a notebook or file that's no git tracked in this repo.
If that's the case, you may want to add a comments block at the top of this file warning fools not to add live DW connections to tests.

Reply:
In `plot_test.py` I included a bunch of tests that I was running locally. These tests are really only useful if they execute the sql, so I added the ability to do so if the snowflake creds are included in .env. I marked this tests as 'skip' because they will fail if running in a github action. I can add some documentation about writing local tests like this, what are your thoughts? I could also just remove the tests, but I thought there was value in adding them even though they need to be run locally with snowflake creds setup.